examples/transfer/plot_rpa_steps.py

```diff
@@ -0,0 +1,109 @@
+"""
+====================================================================
+Plot the data transformations in the Riemannian Procrustes Analysis
+====================================================================
+
+Use the SpectralEmbedding module to plot in 2D the transformations on the data
+points from source and target domains when applying the Riemannian Procrustes
+Analysis [1] to match their statistics.
+
+    .. [1] PLC Rodrigues et al “Riemannian Procrustes analysis: transfer
+    learning for brain-computer interfaces”. IEEE Transactions on Biomedical
+    Engineering, vol. 66, no. 8, pp. 2390-2401, December, 2018
```

Comment:
See how sklearn does it:

https://github.com/scikit-learn/scikit-learn/blob/main/examples/calibration/plot_compare_calibration.py#L197

```suggestion
.. [1] `Riemannian Procrustes analysis: transfer learning for
       brain-computer interfaces <https://ieeexplore.ieee.org/document/8588384>`_,
       PLC Rodrigues, C Jutten, M Congedo
       IEEE Transactions on Biomedical Engineering, vol. 66, no. 8,
       pp. 2390-2401, December, 2018
```

Reply:
Well, you activated my OCD... I updated the reference style in all files of `pyRiemann`...